app/models/couch.rb

```diff
@@ -4,7 +4,7 @@ class Couch < ApplicationRecord
   has_many :nights
   has_many :reservations, through: :nights
 
-  scope :in_city, -> (city) { where(city: city) }
+  scope :in_city, -> (city) { where("lower(city) = ?", city.downcase) }
```

Comment:
Think this is great. Is searching by city the only type of search we'll be doing? Couch name? We could implement some citext postgres magic to remove some of this logic, but if this is the only place, not sure if it's necessary.